### ISSUE TITLE
Extension Request attribute

### ISSUE THREAD

**Issue by donskov** [PeculiarVentures/asn1-schema#34]
Could we add a class for the attribute?

```
extensionRequest ATTRIBUTE ::= {
  WITH SYNTAX ExtensionRequest
  SINGLE VALUE TRUE
  ID pkcs-9-at-extensionRequest
}

ExtensionRequest ::= Extensions
```

Example:

```
303a06092a864886f70d01090e312d302b30290603551d0e04220420da80914ab1c14b4113d5939781fbdc4632793f7794f5a1c96bacf261040f1654
```

**Comment by microshine** [comment]
Is it [RFC2985](https://tools.ietf.org/html/rfc2985)?

We could add `@peculiar/asn1-pkcs9` package for that RFC

**Comment by microshine** [comment]
I published a new package `@peculiar/asn1-pkcs9@2.0.20`

```js
const hex = "303a06092a864886f70d01090e312d302b30290603551d0e04220420da80914ab1c14b4113d5939781fbdc4632793f7794f5a1c96bacf261040f1654";

const attr = AsnConvert.parse(Convert.FromHex(hex), Attribute);
assert.strictEqual(attr.type, id_pkcs9_at_extensionRequest);
assert.strictEqual(attr.values.length, 1);

const extensionRequest = AsnConvert.parse(attr.values[0], ExtensionRequest)
assert.strictEqual(extensionRequest.length, 1);
assert.strictEqual(extensionRequest[0].extnID, id_ce_subjectKeyIdentifier);
```

**Comment by donskov** [comment]
Thanks. Works for me
